Source/SwiftyDropbox/Shared/Handwritten/OAuth/OAuthUtils.swift

```diff
@@ -0,0 +1,28 @@
+///
```

Comment:
OauthUtils+PKCE

Reply:
I will add more util methods that are not PKCE specific.